### PR TITLE
feat(login page): created login page

### PR DESCRIPTION
**Pull request by NatalliaMil**
added fields: email address,password, button - submit/login
related #12

### CODE REVIEW

**Comment by mentortechabc** [comment]
Login URL: 
localhost:5000/login
method POST

File: web/app/login.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Login page</title>
+  </head>
+  <body>
+    <h1>Login</h1>
+    <form action="localhost:5000/login" method="get">
```
Comment (glebhg) [inline web/app/login.html:10]: We can't use form action to call our REST API. We need to use fetch API or AJAX to call backend

File: web/app/login.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <label>E-mail<input type="text" id="username" /></label><br />
```
Comment (glebhg) [inline web/app/login.html:11]: 1. Why type="text" if it has email?

File: web/app/login.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (glebhg) [inline web/app/login.html:1]: Consider to use Material components: 

1. inputs https://material.io/components/text-fields
2. buttons: https://material.io/components/buttons

**Review by glebhg — CHANGES_REQUESTED** [review]

**Review by glebhg — APPROVED** [review]

**Review by yogahub — APPROVED** [review]

**Pull request merged**